src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/PropertyMetadata.cs

```diff
@@ -62,9 +63,11 @@ public PropertyMetadata(IProperty property, Type dbContextType)
 
             IsEnum = property.ClrType.GetTypeInfo().IsEnum;
             IsReadOnly = property.IsReadOnlyBeforeSave;
-
-            IsAutoGenerated = ((property.RequiresValueGenerator
-                || property.IsStoreGeneratedAlways)
+            
+            //TODO: RequiresValueGenerator() is an internal EF Core method.
+            // Need to confirm with EF if this is ok to use. 
+            IsAutoGenerated = ((property.RequiresValueGenerator()
```

Comment:
@bricelam @ajcvickers 
`property.RequiresValueGenerator()` is now in an internal namespace. 
Is it possible for EF to expose it as a public extension method?